test/suite/helper_tiered.py

```diff
@@ -50,42 +50,26 @@ def get_auth_token(storage_source):
 # Get buckets configured for the storage source
 
 # S3 buckets with their regions
-s3_buckets = \
-    [('s3testext', 'ap-southeast-2'),
-    ('s3testext-us', 'us-east-2')]
+s3_buckets = ['s3testext;ap-southeast-2', 's3testext-us;us-east-2']
 
 # Local buckets do not have a region
-local_buckets = \
-    [('bucket1', ''),
-    ('bucket2', '')]
-
-# Get a list of buckets available for the storage source.
-def get_bucket_list(storage_source):
-    if storage_source == 'local_store':
-        return local_buckets
-    if storage_source == 's3_store':
-        return s3_buckets
-    return None
+local_buckets = ['bucket1', 'bucket2']
 
 # Get name of the first bucket in the list.
 def get_bucket1_name(storage_source):
```

Comment:
Instead of defining `get_bucket1_name` and `get_bucket2_name`, can we define `get_bucket_name(storage_source, index)`?

Reply:
As discussed, will be fixed in WT-8855.

Reply:
I had thought so initially, but I don't imagine using more than 2 buckets, and where the function gets called, it looks cleaner and more readable:
```
get_bucket_name('s3_store', 1)
```
vs
```
get_bucket1_name('s3_store')
```

It is hard to figure out what that 1 means without a comment, and it is hard to put too many comments in scenario decit code.